analyzer/evmtokenbalances/evm_token_balances.go

```diff
@@ -151,17 +155,49 @@ func (m Main) getStaleTokenBalances(ctx context.Context, limit int) ([]*StaleTok
 }
 
 func (m Main) processStaleTokenBalance(ctx context.Context, batch *storage.QueryBatch, staleTokenBalance *StaleTokenBalance) error {
-	m.logger.Info("downloading", "stale_token_balance", staleTokenBalance)
-	// todo: assert that token addr and account addr contexts are secp256k1
-	tokenEthAddr, err := client.EVMEthAddrFromPreimage(staleTokenBalance.TokenAddrContextIdentifier, staleTokenBalance.TokenAddrContextVersion, staleTokenBalance.TokenAddrData)
-	if err != nil {
-		return fmt.Errorf("token address: %w", err)
-	}
 	accountEthAddr, err := client.EVMEthAddrFromPreimage(staleTokenBalance.AccountAddrContextIdentifier, staleTokenBalance.AccountAddrContextVersion, staleTokenBalance.AccountAddrData)
 	if err != nil {
 		return fmt.Errorf("account address: %w", err)
 	}
-	if staleTokenBalance.Type != nil {
```

Comment:
if this is no longer possible, could we change staleTokenBalance.Type not to be pointer